src/blocks.php

```diff
@@ -143,7 +35,9 @@ function stackable_register_blocks() {
 				$metadata
 			);
 
-			register_block_type_from_metadata( $metadata['block_json_file'], $register_options );
+			$block_args = array_merge( $metadata, $register_options );
+			$block_type = new WP_Block_Type( $metadata['name'], $block_args );
+			register_block_type( $block_type );
```

Comment:
Any reason why we can't just register the block right away?

```php
register_block_type( $metadata['name'], $block_args );
```

Reply:
based on this [documentation](https://developer.wordpress.org/reference/functions/register_block_type/) , `register_block_type` accepts a string or an instance of `WP_Block_Type` for the first argument, if it's a string it will consider it as a path to the `block.json` and will try to use `register_block_type_from_metadata()` instead.